src/Settings.php

```diff
@@ -126,6 +132,14 @@ public static function parseArguments(array $arguments)
     {
         $arguments = new ArrayIterator(array_slice($arguments, 1));
         $settings = new self;
+        
+        $git = "git show HEAD \
+           --name-only \
+           --diff-filter=ACM \
+           -m \
+           --first-parent \
+           --format=format: \
+           -- "'*' . array('.php', '.phtml', '.php3', '.php4', '.php5')" | egrep -v '^$' || :";
```

Comment:
on second thought, you should probably refactor this to avoid use shell pipe formatting. what is you trying to filter out? can you give example? perhaps fixing --format or other arguments helps

otherwise filter it in php side. more readable